### PR TITLE
Add filetype column to audio and image TSV and db

### LINKED ISSUE #152: [Feature] Add filetype to AudioStore and Audio database
## Problem

Openverse API needs to know the filetype of the main Audio file. We currently don't save it to the database.

## Description

We need to add filetype to the AudioStore, and the Audio database. 

## Alternatives

There is actually no real alternative way of finding out the filetype. For images, it is often possible to get the filetype from the image url extension, but audio urls don't usually have extensions.

## Implementation

- [x] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by obulat**
## Fixes

Fixes #152 by @obulat

## Description

This PR adds filetype column to audio data, because it is not possible to detect it from the URL or in any other way in the API.

## Technical details

This PR is based on #168, and will need to be merged after it

## Checklist

- [ ] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [ ] My pull request targets the *default* branch of the repository (`main` or `master`).
- [ ] My commit messages follow [best practices][best_practices].
- [ ] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: openverse_catalog/dags/provider_api_scripts/test_jamendo.py [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
 from unittest.mock import patch
 
 import jamendo
 from common.licenses.licenses import LicenseInfo
 
 
-RESOURCES = os.path.join(
-    os.path.abspath(os.path.dirname(__file__)), "tests/resources/jamendo"
-)
+RESOURCES = Path(__file__).parent.resolve() / "tests/resources/jamendo"
```
Comment (dhruvkb) [inline openverse_catalog/dags/provider_api_scripts/test_jamendo.py:10]: I didn't know that the `/` operator worked on paths. This is very cool!

**Review by dhruvkb — APPROVED** [review]

File: openverse_catalog/dags/provider_api_scripts/test_jamendo.py (hunk 1 above)
Comment (obulat) [reply openverse_catalog/dags/provider_api_scripts/test_jamendo.py:10]: Another cool part is that `pathlib` automatically converts the separators for other system (eg '/' -> '\' for Windows) !

**Review by zackkrida — APPROVED** [review]

**Comment by krysal** [comment]
After today's discussion, it seems like this column needs to be common to all media types.

**Comment by obulat** [comment]
Closing in favor of #196

**Pull request closed without merging**